carp.common/src/commonMain/kotlin/dk/cachet/carp/common/TimeSpan.kt

```diff
@@ -31,15 +33,37 @@ data class TimeSpan(
          * The value parameter is converted to microseconds, and that number of microseconds is used to initialize the new [TimeSpan].
          * Therefore, value will only be considered accurate to the nearest microsecond.
          */
-        fun fromMilliseconds( ms: Double ): TimeSpan
-        {
-            val ticks: Long = (ms * MICROSECONDS_IN_MS).toLong()
-            return TimeSpan( ticks )
-        }
+        fun fromMilliseconds( ms: Double ): TimeSpan = TimeSpan( (ms * MICROSECONDS_IN_MS).toLong() )
```

Comment:
Why are parameters doubles? They should be integers as well.

Reply:
I'm pretty much just cloning the C# API for this: https://docs.microsoft.com/en-us/dotnet/api/system.timespan.fromseconds?view=netcore-3.1 :)

They use doubles. It is a more flexible API, since you can also specify one minute and a half (`.fromMinutes( 1.5 )`). But more importantly, any output of a calculation doesn't lose precision, and doesn't force you to be converted to microseconds, to construct TimeSpan's for.

Reply:
Maybe I should add overrides for Int's though. I am a bit annoyed by having to write `.o` in `.fromMinutes( 15.0 )`.